### DISCUSSION TITLE
Unable to connect two SAP Hana DB Data Lake

### DISCUSSION THREAD

**Discussion by marcrupprath** [dbeaver/dbeaver#35778] [Q&A] [answered]
### Description

Hello,
i managed to sucessfully establish a connection two an SAP Hana DB cloud instance but not two the coresponding 
SAP HANA DB data Lake.

I got the follwing Error message:
SAP DBTech JDBC: Cannot connect to jdbc:sap://3023f74f-249d-4325-8079-7b770216e54c.iq.hdl.trial-us10.hanacloud.ondemand.com:443 [Object is closed: com.sap.db.jdbc.SecureChannelSession@5d475444 3023f74f-249d-4325-8079-7b770216e54c.iq.hdl.trial-us10.hanacloud.ondemand.com:443 ConnectionID:0 SessionID:0 on sun.nio.ch.WindowsAsynchronousSocketChannelImpl[connected local=/[0:0:0:0:0:0:0:0]:59708 remote=3023f74f-249d-4325-8079-7b770216e54c.iq.hdl.trial-us10.hanacloud.ondemand.com/52.23.3.236:443]: w.x.y.z.z:59708 -> 3023f74f-249d-4325-8079-7b770216e54c.iq.hdl.trial-us10.hanacloud.ondemand.com:443 ConnectionID:0 SessionID:0].
  
Note: w.x.y.z.z = my local IP adress within our companies LAN 

The SQL connection settings two db that works is like:
![2024-04-15_Working_Connection_with_SAp_Cloud_HANA_DB](https://github.com/dbeaver/dbeaver/assets/6498534/f4d2dd6b-6cbd-4fc9-8860-d63fba05efc4)

The connection that does not work to SAP Hana Cloud DB Data Lake

![2024-04-15_Non_Working_Connection_with_SAP_Cloud_HANA_DB_DATALAKE](https://github.com/dbeaver/dbeaver/assets/6498534/b3935c4b-cf9c-4645-a985-bc4cd62403ca)

I asked myself what wen't wrong with connection to DATA LAke ?

Thanks in advance

### DBeaver Version

Community Version 24.0.2.202404071654

### Operating System

Windows 10 22H2

**Comment by E1izabeth** [comment]
Probably, we don't support SAP HANA DB Data Lake connection

**Comment by kai-morich** [comment, marked as answer]
*HANA Cloud Data Lake Relational Engine* is based in SAP IQ (formerly known as Sybase IQ) technology. 
Please look [here](https://developers.sap.com/group.hana-cloud-clients-data-lake.html) to figure out the right ODBC or JDBC driver and [here](https://dbeaver.com/docs/dbeaver/Database-drivers/) on how to configure in DBeaver.

**Comment by E1izabeth** [comment]
@marcrupprath, did the suggested advice help you?

**Comment by marcrupprath** [comment]
@E1izabeth : Yes i have gotten a Driver from a 3-rd party and it then works fine with DBeaver.

**Comment by syy6** [comment]
jConnect driver in DBeaver can work with SAP HANA Datalake, just needs to setup the driver paramter
as
CHARSET:utf8
ENABLE_SSL:true
